CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the chef-client cookbook.
 
+## 7.3.0 (2017-03-13)
```

Comment:
Please don't update the CHANGELOG (or metadata). This is more likely to cause conflicts if other PRs come in and get pulled and the cookbook gets released.